notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<p>Could you add a small discussion/results block -- to summarise what we find from the analysis? This is for us when we are bringing all results together to write our grand technote! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
But smartly written notebook -- I learned a lot of tricks about how to pull EFD data smartly :D This notebook will be my reference!

Reply:
<p>I added a small summary of the findings. I think further analysis and conclusions warrant looking at the whole picture.</p>